RickAndMorty-iOS/Views/Episodes/EpisodesViewController.swift

```diff
@@ -0,0 +1,128 @@
+//
+//  EpisodesViewController.swift
+//  RickAndMorty-iOS
+//
+//  Created by Gagan on 2023-03-14.
+//
+
+import UIKit
+import Combine
+
+class EpisodesViewController: UIViewController {
+
+    let episodesView = EpisodesView()
+    let viewModel = EpisodesViewModel()
+    weak var coordinator: MainCoordinator?
```

Comment:
This is wrong according to a MVVM. ViewController should not communicate with coordinator